servers/src/main/java/tachyon/master/MasterContext.java

```diff
@@ -55,10 +55,19 @@ public static MasterSource getMasterSource() {
   }
 
   /**
-   * Reset the TachyonConf instance in master context, for test only.
+   * Resets the master context, for test only.
```

Comment:
Agree with avoiding using public methods only for testing. But I wish to address this in a separate PR (and try a different approach from whitebox). My concern with simply using whitebox in this particular case is that the re-initialize logic will be moved to the integration test which might be miss match in the future if we add new stuff into context class but forget to update the integration tests.